.github/workflows/fabric_acctest.yml

```diff
@@ -116,7 +116,16 @@ jobs:
           EQUINIX_API_CLIENTSECRET: ${{ secrets.EQUINIX_API_CLIENTSECRET_PNFV }}
           METAL_AUTH_TOKEN: ${{ secrets.METAL_AUTH_TOKEN }}
         run: |
-          go test ./... -v -coverprofile coverage_pnfv.txt -covermode=atomic -count 1 -parallel 8 -run "(PNFV)" -timeout 180m
+          go test ./... -v -coverprofile coverage_pnfv.txt -covermode=atomic -count 1 -parallel 8 -run "(PNFV)" -timeout 180m | tee pnfv_test_output.log
+
+      - name: Upload PNFV Testing Log
+        uses: actions/upload-artifact@v4
+        with:
+          name: pnfv_test_logs
+          path: pnfv_test_output.log
+
+      - name: Check if Tests are passed
+        run: sh scripts/check_tests.sh pnfv_test_output.log
```

Comment:
Test runs are failing in part because this script file doesn't exist in the repo.  Can you add it?